src/servo_pkg/servo_pkg/servo_client.py

```diff
@@ -0,0 +1,81 @@
+import rclpy
+from rclpy.node import Node
+from interfaces.srv import MoveServo
+
+import time
+
+
+class Servo_Client(Node):
+    def __init__(self):
+        super().__init__("servo_Client")
+
+        self.cli = self.create_client(MoveServo, "turn_servo")
+        while not self.cli.wait_for_service(timeout_sec=1.0):
+            self.get_logger().info("service not available, waiting again...")
+        self.req = MoveServo.Request()
+
+    def send_request(self, port: int, pos: int) -> MoveServo:
+        self.req.port = port
+        self.req.pos = pos
+        self.future = self.cli.call_async(self.req)
+        rclpy.spin_until_future_complete(self, self.future)
+        return self.future.result()
+
+    def USB_request(self) -> None:
+        # Move servo on port 0 to minimum position
+        servo1 = MoveServo.Request()
+        servo1.port = 0
+        servo1.pos = 2432
+
+        response = Servo_Client.send_request(self, servo1.port, servo1.pos)
```

Comment:
```suggestion
        response = Servo_Client.send_request(port=0, pos=2432)
```
Pretty sure this will do what we need